components/H1Styled/H1Styled.styles.js

```diff
@@ -0,0 +1,40 @@
+import { H1 } from '../../style/typography';
+import styled from 'styled-components';
+import { min } from '../../lib/responsive';
+
+const H1Container = styled.div`
+  white-space: nowrap;
+  @media ${min.mobile} {
+    margin: 25em 0 5em 7.5em;
+  }
+  @media ${min.tablet} {
+    margin: 20.5em 0 4em 8em;
+  }
+  @media ${min.desktop} {
+    margin: 4em 0 2em 5.9em;
+  }
+`;
+
+const H1Regular = styled(H1)`
+  margin: 0 auto;
+`;
+
+const H1Italicized = styled(H1)`
```

Comment:
I think we can pull `padding: 0` out of the @media because it's consistent in all